static/docs/commands-reference/pull.md

```diff
@@ -200,4 +200,3 @@ the `model.p.dvc` stage occurs later, its data was not pulled.
 Then we ran `dvc pull` specifying the last stage, `model.p.dvc`, and its data
 was downloaded. Finally, we ran `dvc pull` with no options to make sure that all
 data was already pulled with the previous commands.
-
```

Comment:
Please don't remove line breaks at the end of files. If this is your IDE or code editor please configure it to not do this.